### PR TITLE
OPHJOD-616: Update background color of RoundButton & RoundLinkButton

### PR DESCRIPTION
**Pull request by ketsappi**
## Description

Update background color of RoundButton & RoundLinkButton.
Fix the hover; was able to hover label and circle shape separately.

### Before
SCR-20240815-ikfp

### After
SCR-20240815-irda

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-616

### CODE REVIEW

File: lib/components/RoundLinkButton/RoundLinkButton.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const RoundLinkButton = ({
     >
       <span
         aria-hidden
-        className={`${selected ? 'text-white' : 'text-black'} material-symbols-outlined size-48 group- flex size-[64px] select-none items-center justify-center self-center rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray hover:text-accent'}`}
+        className={`${selected ? 'text-white' : 'text-black'} material-symbols-outlined size-48 group- flex size-[64px] select-none items-center justify-center self-center rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray-2 hover:text-accent'}`}
```
Comment (juhaniko) [inline lib/components/RoundLinkButton/RoundLinkButton.tsx:40]: Could these ternaries be combined? `selected ? 'text-white bg-accent' : 'text-black bg-bg-gray'`

File: lib/components/RoundButton/RoundButton.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const RoundButton = ({
     >
       <span
         aria-hidden
-        className={`size-[72px] rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray'} flex items-center justify-center ${selected ? 'text-white' : 'text-black hover:text-accent'} material-symbols-outlined size-48 select-none`}
+        className={`size-[72px] rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray-2'} flex items-center justify-center ${selected ? 'text-white' : 'text-black hover:text-accent'} material-symbols-outlined size-48 select-none`}
```
Comment (juhaniko) [inline lib/components/RoundButton/RoundButton.tsx:38]: Same thing as the other comment below, combine ternaries: `selected ? 'bg-accent text-white' : 'bg-bg-gray-2 text-black'`

File: lib/components/RoundLinkButton/RoundLinkButton.tsx (hunk 1 above)
Comment (ketsappi) [reply lib/components/RoundLinkButton/RoundLinkButton.tsx:40]: Yeah, sure. I can do in this ticket while at it.

Comment (ketsappi) [reply lib/components/RoundLinkButton/RoundLinkButton.tsx]: This is now changed. Also noticed the bug with hovering; was able to hove label and the circle shape separately. Fixed to now have only one hover.

**Review by juhaniko — APPROVED** [review]

**Pull request merged**